apps/meteor/client/hooks/notification/useNotification.ts

```diff
@@ -63,54 +63,56 @@ export const useNotification = () => {
 			n.close();
 			window.focus();
 
-			const jump = msgId && { jump: msgId };
+			if (!notification.payload._id || !notification.payload.rid || !notification.payload.name) {
```

Comment:
You removed _id and name from the destructing but still being in use here.

Also, rid is already destructured and you're accessing it from the full path.